cmds/core/find/find.go

```diff
@@ -18,84 +18,113 @@ import (
 	"context"
 	"flag"
 	"fmt"
+	"io"
 	"log"
 	"os"
 	"strings"
 
 	"github.com/u-root/u-root/pkg/find"
 )
 
-const cmd = "find [opts] starting-at-path"
+type params struct {
+	fileType string
+	name     string
+	perm     int
+	long     bool
+	debug    bool
+}
 
-var (
-	perm     = flag.Int("mode", -1, "Permissions")
-	fileType = flag.String("type", "", "File type")
-	name     = flag.String("name", "", "glob for name")
-	long     = flag.Bool("l", false, "long listing")
-	debug    = flag.Bool("d", false, "Enable debugging in the find package")
-)
+type cmd struct {
+	stdout io.Writer
+	strerr io.Writer
+	args   []string
+	params params
+}
+
+func command(stdout, stderr io.Writer, args []string, params params) *cmd {
```

Comment:
generally, I'd expect the []string to be last. I don't know, must a style thing I'm used to.

Reply:
done